inst/templates/get-started.Rmd

```diff
@@ -383,6 +383,49 @@ vroom::vroom_write(to_edit, "to_edit.csv", delim = ",")
 writexl::write_xlsx(to_edit, "to_edit.xlsx")
 ```
 
+### Calculate coverage statistics
+
+```{r}
+# Matched companies based on suggested matches
+matched_rows <- to_edit %>%
+  filter(suggest_match) %>%
+  nrow()
+
+# Share of matched companies from total loanbook companies
+matched_share <- to_edit %>%
+  filter(suggest_match) %>%
+  nrow() / loanbook %>%
+    distinct(company_name) %>%
+    nrow()
+matched_share
+
+# Number and share of matched companies classified by misc_info from total loanbook companies
+# misc_info should be replaced with variables like sectors, headcount etc.
+misc_share <- loanbook %>%
+  count(misc_info) %>%
+  left_join(to_edit %>%
+    filter(suggest_match) %>%
+    count(misc_info), by = c("misc_info"), suffix = c("_loanbook", "_merged")) %>%
```

Comment:
Maybe create independently each of the `x` and `y` objects that you'll pass to `left_join()`. Creating the `y` object with a long pipe inside `left_join()` is hard to read.

A clear pattern is this:

```r
x <- data %>% ...
y <- data %>% ...
left_join(x, y)
```